packages/synapse-react-client/src/lib/containers/auth/TOTPForm.tsx

```diff
@@ -27,6 +27,10 @@ export default function TOTPForm(props: TOTPFormProps) {
         }}
         gap={0}
         sx={{
+          '.MuiInputBase-root': {
+            paddingLeft: '5px',
+            paddingRight: '5px',
+          },
```

Comment:
The TOTP form (6 single-digit inputs) is a special case where the default theme InputBase styles should be overridden